### ISSUE TITLE
Backend status

### ISSUE THREAD

**Issue by fmorency** [liftedinit/many-ghostcloud#9]
As a user, I want to be able to see the backend status from any page.

**Comment by nblenke** [comment]
Could you please provide an example of this?

**Comment by fmorency** [comment]
The backend supports the `heartbeat` endpoint. We could probe that endpoint every 30s or so and display the corresponding status this way.

I can show you how to build and run the backend locally during a 1:1.

**Comment by nblenke** [comment]
That sounds reasonable, whats the format of the endpoints? Totally guessing here but these return a 500:
```
curl --location --request GET 'http://127.0.0.1:8000/heartbeat'
curl --location --request GET 'http://127.0.0.1:8000/api/heartbeat'
```

**Comment by fmorency** [comment]
> That sounds reasonable, whats the format of the endpoints? Totally guessing here but these return a 500:
> 
> ```
> curl --location --request GET 'http://127.0.0.1:8000/heartbeat'
> curl --location --request GET 'http://127.0.0.1:8000/api/heartbeat'
> ```

I wish it was that simple... [1]

You need to use the `heartbeat` function of the `Base` module [2]. You _should_ be able to call it using something like

```
const [activeNetwork, ,] = useNetworkContext()
await activeNetwork?.base?.heartbeat()
```

[1]: It will be in our next project!
[2]: https://github.com/liftedinit/many-js/blob/11faaf92a61bca4252d0d0f5d5b768b5d46f65cc/src/network/modules/base/base.ts#L38

**Comment by nblenke** [comment]
Very Nice! I am noticing that there's no data in the response for `await activeNetwork?.base?.heartbeat()`, I do see data on `await activeNetwork?.base?.status()`. Is that expected?

**Comment by fmorency** [comment]
> Very Nice! I am noticing that there's no data in the response for `await activeNetwork?.base?.heartbeat()`, I do see data on `await activeNetwork?.base?.status()`. Is that expected?

Yes, this is expected, as seen in https://github.com/many-protocol/specification/blob/df6c8afa4ba709a7263353f983f2000d7e727da6/attributes/network/cddl/0_base.cddl#L49-L52

**Comment by fmorency** [comment]
Closed by #22
